setup.py

```diff
@@ -24,6 +24,8 @@
                    "toposort >= 1.6", "wheel >= 0.36.2",
                    "DBUtils >= 2.0",
                    "sqlalchemy >= 1.4.17",
+                   "structlog >= 21.1.0",
+                   "colorama >= 0.4.4",
```

Comment:
Can you tell me more about why we are adding colorama?

Reply:
It allows for colorful console output, if someone wants to use that. We can remove it you think it is overkill.

Reply:
In the interest of simplicity, I will take out the colorama requirement and the configuration that would use it.